Added PowerShell cmdlets for getting offline token

JanVidarElven opened the pull request:
Updated Application Proxy silent install with PowerShell cmdlets for getting offline token for registering connector

azurecla commented:
Hi __@JiangChen79__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (cjiang). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

JiangChen79 commented:
#sign-off

tysonn commented:
@JiangChen79 - Please see pull request https://github.com/Azure/azure-content-pr/pull/17281 in the private repo to review validation results and the staged version of your updates.

JanVidarElven committed:
Added PowerShell cmdlets for getting offline token

Updated Application Proxy silent install with PowerShell cmdlets for getting offline token for registering connector
